tools/configen/configen/configen.py

```diff
@@ -54,9 +56,7 @@ def init_config(conf_dir: str) -> None:
 def save(cfg: ConfigenConf, module: str, code: str) -> None:
     module_path = module.replace(".", "/")
 
-    module_path_pattern = Template(cfg.module_path_pattern).render(
-        module_path=module_path
-    )
+    module_path_pattern = Template(cfg.module_path_pattern).render(module_path=module_path)
```

Comment:
Did buck change it's mind about this or are you using some other buck configuration?

Reply:
automatic black reformatting using my default configuration :]